satpy/etc/enhancements/generic.yaml

```diff
@@ -1230,3 +1230,60 @@ enhancements:
          stretch: crude
          min_stretch: [0,0,0]
          max_stretch: [1,1,1]
+
+  Cloud_Top_Height:
+    standard_name: cldprop_cloud_top_height
+    operations:
+      - name: colorize
+        method: !!python/name:satpy.enhancements.colorize
+        kwargs:
+          palettes:
+          - {colors: [[255, 0, 0], [255, 0, 0]], min_value: 0, max_value: 800}
+          - {colors: [[170, 0, 0], [170, 0, 0]], min_value: 800.0001, max_value: 1600}
+          - {colors: [[110, 0, 0], [110, 0, 0]], min_value: 1600.0001, max_value: 2350}
+          - {colors: [[112, 1, 2], [112, 1, 2]], min_value: 2350.0001, max_value: 3150}
+          - {colors: [[124, 91, 5], [124, 91, 5]], min_value: 3150.0001, max_value: 4000}
+          - {colors: [[240, 190, 64], [240, 190, 64]], min_value: 4000.0001, max_value: 4800}
+          - {colors: [[255, 255, 0], [255, 255, 0]], min_value: 4800.0001, max_value: 5600}
+          - {colors: [[0, 220, 0], [0, 220, 0]], min_value: 5600.0001, max_value: 6400}
+          - {colors: [[0, 136, 0], [0, 136, 0]], min_value: 6400.0001, max_value: 7200}
+          - {colors: [[0, 80, 0], [0, 80, 0]], min_value: 7200.0001, max_value: 8000}
+          - {colors: [[0, 136, 238], [0, 136, 238]], min_value: 8000.0001, max_value: 8800}
+          - {colors: [[0, 0, 255], [0, 0, 255]], min_value: 8800.0001, max_value: 9600}
+          - {colors: [[0, 0, 170], [0, 0, 170]], min_value: 9600.0001, max_value: 10400}
+          - {colors: [[0, 0, 100], [0, 0, 100]], min_value: 10400.0001, max_value: 11200}
+          - {colors: [[183,15,141], [183, 15, 141]], min_value: 11200.0001, max_value: 12000}
+          - {colors: [[102, 0, 119], [102, 0, 119]], min_value: 12000.0001, max_value: 18000}
+
+  Clear_Sky_Confidence:
+    standard_name: cldmsk_clear_sky_confidence
+    operations:
+      - name: colorize
+        method: !!python/name:satpy.enhancements.colorize
+        kwargs:
+          palettes:
+          - {
+             filename: clear_sky_confidence.txt
```

Comment:
Does Satpy find this colormap in `etc/colormaps/`?

Reply:
I set the config path to the etc/colormaps/ directory locally before running any of this

Reply:
Which config path?

Reply:
I set SATPY_CONFIG_PATH using `satpy.config.set(config_path=['xxx/etc/colormaps/'])`

Reply:
Could you try not modifying the config path and specify these in the YAML as `colormaps/clear_sky_confidenct.csv` and so on?

Reply:
That works as well, just updated.